test/add-member.test.js

```diff
@@ -54,7 +55,10 @@ test('add member', async (t) => {
     const group = await kaitiaki.tribes2.create()
     t.true(group.id, 'creates group')
 
-    const authorIds = [newPerson.id, ssbKeys.generate().id]
+    const authorIds = [
+      fromFeedSigil(newPerson.id),
+      fromFeedSigil(ssbKeys.generate().id),
+    ]
```

Comment:
:hot_pepper: I think you want to test it can take one of each and not panic right?

Your code I read I think maps them into URIs anyway right?